recipes/coincurve/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set name = "coincurve" %}
+{% set version = "18.0.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c86626afe417a09d8e80e56780efcae3ae516203b23b5ade84813916e1c94fc1
+  patches:
+    - resolve_abs_path_SOURCES.txt.patch  # [not win]
+    - update_pyproject.toml.patch
+    - update_setup.py.patch  # [win]
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - /lib64/libc.so.6
+  script_env:
+    - COINCURVE_UPSTREAM_REF=ddf2b2910eb19032f8dd657c66735115ae24bfba
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pip
+    - python
+    - python-build
+    - requests
+    - setuptools
+    - wheel
+    - autoconf  # [not win]
+    - automake  # [not win]
+    - libtool  # [not win]
+    - m4  # [not win]
+    - make  # [not win]
+    - pkg-config  # [not win]
+    - asn1crypto  # [win]
+    - m2-bash  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-gzip  # [win]
+    - m2-libtool  # [win]
+    - m2-m4  # [win]
+    - m2-pkg-config  # [win]
+    - m2-sed  # [win]
+    - m2w64-gcc  # [win]
+    - m2w64-make  # [win]
+    - posix  # [win]
+    - pydflow  # [win]
+
+  host:
+    - asn1crypto
+    - cffi >=1.3.0
+    - python
+    - requests
+    - pip
+
+  run:
+    - asn1crypto
+    - cffi >=1.3.0
+    - python
+
+test:
+  imports:
+    - coincurve
+  files:
+    - tests
+  requires:
+    - pip
+    - coverage
+    - pytest
+    - pytest-benchmark
+  commands:
+    - pip check
+    - pytest
+
+about:
+  home: https://github.com/ofek/coincurve
+  summary: Cross-platform Python CFFI bindings for libsecp256k1
+  doc_url: https://ofek.dev/coincurve/
+  description: |
+    This library provides well-tested Python bindings for libsecp256k1, 
```

Comment:
It looks like this package is redistributing https://github.com/bitcoin-core/secp256k1. This is not allowed. secp256k1 should be build as a separate conda package and dynamically linked.

This package seems to have support for using system installed secp256k1.
https://github.com/ofek/coincurve/blob/2dfea673835e524ffaeb11dc1c2aecaf23188195/setup_support.py#L74C8-L74C8

Reply:
> What is the typical merge time for a PR.

Reviewers are unpaid volunteers, so there is not typical amount of time.

> but only import the minimum from the libsecp256k1 recipe.

I don't understand what you mean.

Reply:
> Reviewers are unpaid volunteers, so there is not typical amount of time.
Of course, I just wanted to understand what strategy I should use bundling or not, no other intent. Another way to ask is as a reviewer, do you prefer to review a bundle of 5+ packages or singled-out recipes?

> I don't understand what you mean.
The part of `libsecp256k1` that I included was a modified version that only had the parts that I needed for `coincurve` to build successfully on all the platforms

So the plan forward is to merge `libsecp256k1` to this recipe, do we agree?
I moved BIP to work in progress and will wait until `coincurve` is merged?

Reply:
> Of course, I just wanted to understand what strategy I should use bundling or not, no other intent. Another way to ask is as a reviewer, do you prefer to review a bundle of 5+ packages or singled-out recipes?

Packages should be bundled into related groups of around 5-ish packages or less.

> The part of libsecp256k1 that I included was a modified version that only had the parts that I needed for coincurve to build successfully on all the platforms So the plan forward is to merge libsecp256k1 to this recipe, do we agree?

This single PR should contain multiple recipes such that libsecp256k1 is separate from coincurve. Coincurve recipe can depend on the output of the libsecp256k1 recipe.

> I moved BIP to work in progress and will wait until coincurve is merged?

Perhaps wait until your first PR is merged before working on the others, so that you have more experience.

Reply:
Separated `libsecp256k1`. Resolved